tests/integration_tests.rs

```diff
@@ -0,0 +1,384 @@
+use std::env::VarError;
+use std::io::ErrorKind;
+use std::net::{SocketAddr, TcpStream, ToSocketAddrs};
+use std::path::PathBuf;
+use std::process::{Child, Stdio};
+use std::str;
+use std::thread::sleep;
+use std::time::{Duration, Instant};
+use std::{env, fs};
+
+use assert_cmd::Command;
+use indent::indent_by;
+use predicates::prelude::PredicateBooleanExt;
+use serde_json::Value;
+use static_init::dynamic;
+use tempfile::{NamedTempFile, TempPath};
+use uuid::Uuid;
+
+#[test]
+fn without_command_usage_is_shown() {
+    Command::cargo_bin("peridio-cli")
+        .unwrap()
+        .assert()
+        .code(2)
+        .stderr(predicates::str::contains(
+            "Usage: peridio-cli [OPTIONS] <COMMAND>",
+        ))
+        .stderr(
+            predicates::str::contains("-a, --api-key <API_KEY>")
+                .and(predicates::str::contains("[env: PERIDIO_API_KEY]")),
+        )
+        .stderr(
+            predicates::str::contains("-b, --base-url <BASE_URL>")
+                .and(predicates::str::contains("[env: PERIDIO_BASE_URL=]")),
+        )
+        .stderr(
+            predicates::str::contains("-c, --ca-path <CA_PATH>")
+                .and(predicates::str::contains("[env: PERIDIO_CA_PATH=]")),
+        )
+        .stderr(
+            predicates::str::contains("-o, --organization-name <ORGANIZATION_NAME>").and(
+                predicates::str::contains("[env: PERIDIO_ORGANIZATION_NAME=]"),
+            ),
+        )
+        .stderr(
+            predicates::str::contains("-p, --profile <PROFILE>")
+                .and(predicates::str::contains("[env: PERIDIO_PROFILE=]")),
+        )
+        .stderr(
+            predicates::str::contains("-d, --config-directory <CONFIG_DIRECTORY>").and(
+                predicates::str::contains("[env: PERIDIO_CONFIG_DIRECTORY=]"),
+            ),
+        );
+}
+
+#[test]
+fn with_users_subcommands_are_shown() {
+    Command::cargo_bin("peridio-cli")
+        .unwrap()
+        .arg("users")
+        .assert()
+        .code(2)
+        .stderr(predicates::str::contains(
+            "Usage: peridio-cli users <COMMAND>",
+        ))
+        .stderr(predicates::str::contains("  me"));
+}
+
+#[test]
+fn with_users_with_me_shows_email_and_username() {
+    let base_url = base_url();
+    let ca_path_buf = peridio_cloud_certificate_authority_path();
+    let user = User::create(&format!("{}.com", random_name()));
+    let api_key = user.create_api_key();
+    let ca_path = ca_path_buf.into_os_string().into_string().unwrap();
+
+    PERIDIO_CLOUD_API.init();
+
+    let assert = Command::cargo_bin("peridio-cli")
+        .unwrap()
+        .args(&["--base-url", &base_url])
+        .args(&["--ca-path", &ca_path])
+        .args(&["--api-key", &api_key])
```

Comment:
I don't think you need to reference `&` the array/list argument on those 3 places

Reply:
It is necessary to change the type from `String` to `&str`
![Screenshot 2024-02-27 at 10 17 17 AM](https://github.com/peridio/peridio-cli/assets/298259/21351c27-3c90-44ac-8607-b30b7ea08fb3)

Reply:
Fixed